src/ActionDropdown/List.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { FC } from 'react'
+import styled from 'styled-components'
+
+import { theme } from '../theme'
+
+export type ActionListItem = {
+  label: string
+  value: string
```

Comment:
Could this be a string literal to address my earlier comments?

Reply:
Answering to both comments here:

In order to avoid narrowing down the potential use cases for this component, I left the type definition quite generic.

It is up to the user to define what the _label-value_ pairs would look like.

e.g:
```js
{ label: "I am a label", value: "value"  }
or
{ label: "NoT RelAteD to Th3 Val", value: "foobar" }
```
@alexwllms